### ISSUE TITLE
Tabellenformatierung: Farbe der Tabellenlienien nicht veränderbar

### ISSUE THREAD

**Issue by an-mus** [jlawyerorg/j-lawyer-org#583]
Die Einstellung der Tabellenlinienfarbe funktioniert noch nicht. 
Wenn die Tabelle über die Zwischenablage exportiert wird verändert sich die Farbe nicht. Wird das Dokument direkt erstellt, wird nur die rechte Spalte korrekt gefärbt.

@j-dimension : mir ist noch nicht ganz klar, ob der Fehler in den Plugins oder unter: jlawyer.plugins.calculation.StyledCalculationTable liegt.

**Comment by j-dimension** [comment]
für HTML ist es ein Fehler in der StyledCalculationTable, für die LibreOffice-Dokumente augenscheinlich ein Fehler in der von j-lawyer.org verwendeten Bibliothek (ODF Toolkit). Ich werde im Rahmen der Version 1.12 zunächst auf die neueste Version dieser Bibliothek wechseln. Ticket bleibt daher offen.

**Comment by an-mus** [comment]
@j-dimension : verändert hat sich was. In Version 1.13 unter Linux funktioniert es beim Export über die Zwischenablage. Über die Dokument erstellen Funktion wird die Linienfarbe nicht geändert ...

**Comment by j-dimension** [comment]
> Über die Dokument erstellen Funktion wird die Linienfarbe nicht geändert ...

@an-mus Genau, da muss ich noch etwas tun. Das Ticket ist aktuell für die 1.15 eingeplant. Brauchst Du es früher?

**Comment by an-mus** [comment]
Nein keine Eile. Ich bin nur gestern die offenen Tickets durchgegangen und war über dieses gestolpert. Ich würde sagen unterste Priorität unter den Anpassungspunkten für die StyledCalculationTable.
Meinetwegen können wir das Feature "Farbe der Tabellenlinie" auch raus werfen, bis sich das jemand wünscht. Wenn du das auch so siehst schließ einfach das Ticket. Ich würde dann die Funktionen aus den Plugins entfernen.

**Comment by j-dimension** [comment]
@an-mus ich lass es mal noch offen - bin guter Dinge dass es in die 1.15 reinpassen wird.

**Comment by j-dimension** [comment]
Bibliothekupdate hat nichts gebracht, nochmal probieren nachdem #1472 bearbeitet ist.
